code/modules/atmospherics/portable/canister.dm

```diff
@@ -498,8 +497,8 @@
 	if (!src.holding && !src.connected_port)
 		logTheThing("station", usr, null, "[valve_open ? "opened [src] into" : "closed [src] from"] the air [log_atmos(src)] at [log_loc(src)].")
 		playsound(src.loc, "sound/effects/valve_creak.ogg", 50, 1)
-		playsound(src.loc, "sound/machines/hiss.ogg", 50, 1)
 		if (src.valve_open)
+			playsound(src.loc, "sound/machines/hiss.ogg", 50, 1)
```

Comment:
moved this so people can tell from the sound alone whether it was opened or closed